src/bake/DevServer.zig

```diff
@@ -2411,7 +2411,12 @@ pub fn finalizeBundle(
         if (dev.has_tailwind_plugin_hack) |*map| {
             const first_1024 = code.buffer[0..@min(code.buffer.len, 1024)];
             if (std.mem.indexOf(u8, first_1024, "tailwind") != null) {
-                try map.put(dev.allocator, key, {});
+                const gop = try map.getOrPut(dev.allocator, key);
+                if (!gop.found_existing) {
+                    // Get a stable key from client_graph that will outlive the bundler heap
+                    const stable_key = (try dev.client_graph.insertEmpty(key, .css)).key;
+                    gop.key_ptr.* = stable_key;
```

Comment:
```suggestion
                    gop.key_ptr.* = try dev.allocator.dupe(u8, key);
```
I think it should be just be a clone instead, similar to how it's done in `receiveChunk` and `insertCssFileOnServer`. Same problem there: both functions are called in a scope that defers `graph.heap.deinit`, so the key string needs to be cloned.

Then on line 762:
```diff
        .has_tailwind_plugin_hack = if (dev.has_tailwind_plugin_hack) |*hack| {
+           for (hack.keys()) |key| {
+                dev.allocator.free(key);
+           }
            hack.deinit(allocator);
        },
```

Reply:
@dylan-conway I changed it to follow this battern 
<img width="820" alt="image" src="https://github.com/user-attachments/assets/9b6f5eaa-3ef3-4851-9b13-5bc6ae65d629" />

Reply:
@dylan-conway want me to commit your suggestion?

Reply:
@190n @dylan-conway the dupe / clone approach was the initial route, should I commit this change?